### ISSUE TITLE
Latest version on Hackage and GitHub differ

### ISSUE THREAD

**Issue by ScottG489** [hadolint/hadolint#855]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
The versions listed on Hackage and GitHub would be the same.

### Actual behavior
Hackage lists the latest version as 2.11.0, but GitHub lists it as 2.10.0

### Steps to reproduce the behavior

N/A

**Additional environment details (OS, stack version, etc.)**
This isn't a big deal, but thought you might like to know there's a slight discrepancy. I took a quick look at the Actions and `docs/RELEASE.md`, but couldn't figure out why this might be. The docs in particular mention:
```sh
export TAG=v1.2.3 && git pull && git tag -a "$TAG" -m "$TAG" && git push origin master --tags && stack upload .
```
but running that would technically update GitHub first before Hackage.

It looks like there was an issue with the latest Actions workflow run which may be a culprit: https://github.com/hadolint/hadolint/actions/runs/2641869230

**Comment by lorenzo** [comment]
Yes, it will take a while to solve the issue. Hopefully in the next couple of days.

**Comment by m-ildefons** [comment]
I'll close this issue since it has been solved now.
The release workflow takes care of releasing on Hackage, Dockerhub and GitHub automatically, keeping all three places in sync.
